examples/custom-network-config/Taskfile.yml

```diff
@@ -5,10 +5,10 @@ dotenv:
 silent: false
 
 env:
-  FST_CHART_VERSION: v0.30.0
+  SOLO_CHART_VERSION: v0.30.0
```

Comment:
```suggestion
  SOLO_CHART_VERSION: v0.32.0
```

Reply:
or should it be v0.33.0?

I noticed that there is a v0.33.0 for both charts and containers, but the release tag shows v0.32.0 and the v0.33.0 is missing from the releases.